tests/test_validators.py

```diff
@@ -529,7 +473,7 @@ def test_authors_titles_validator_fails_authors_and_titles_match_title_but_not_a
             ],
             'titles': [
                 {
-                    'title': 'a matching title'  # 0.0 < Jaccard index < 0.5
+                    'title': 'a matching title'
```

Comment:
As a general comment, these tests are written too closely with the current implementation in mind: no record in the world has this many titles so different from one another, so this test won't act as a regression test for whatever implementation we might write for this validator, but is just specifying precisely what the current implementation does and does not.